cue.go

```diff
@@ -94,7 +94,7 @@ func CueValidate(query, cueFile, currentPath string) (tc CanBeAPart, err error)
 		ptc, _ := t.Validate(rootValue, CuePath{}, blockedRootFields)
 		tc = ptc
 		if ptc.Error != nil {
-			err = fmt.Errorf(*ptc.Error)
+			err = fmt.Errorf("%s", *ptc.Error)
```

Comment:
We actually don't need fmt.Errorf at all here. 
```suggestion
			err = *ptc.Error
```

Reply:
ptc.Error apparently is a *string

Reply:
Ok no worries.